pkgs/by-name/si/simdutf/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, libiconv
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "simdutf";
+  version = "4.0.5";
+
+  src = fetchFromGitHub {
+    owner = "simdutf";
+    repo = "simdutf";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-HNTVo/uB7UTCy5VVdmf6vka9T+htra7Vk7NF4hByGP4=";
+  };
+
+  # Fix build on darwin
+  postPatch = ''
+    substituteInPlace tools/CMakeLists.txt --replace "-Wl,--gc-sections" ""
+  '';
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
```

Comment:
```suggestion
  buildInputs = lib.optionals stdenv.isDarwin [
```

Reply:
libiconv is also used in linux, but it stemming from the C library, so add it to buildInputs is not necessary but should acceptable

```
/nix/store/w9mn7nhrd36xrsa8pn141ad0azkkhbxa-glibc-iconv-2.38
└── include
   └── iconv.h -> /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/iconv.h
```